shell.nix

```diff
@@ -29,17 +35,22 @@ in
       pkg-config
       pixman cairo giflib libjpeg libpng librsvg pango            # build dependencies for node-canvas
       shellcheck
+      openssl # for LD_LIBRARY_PATH > WabiSabiClientLibrary
```

Comment:
You don't need to add stuff here if you need it only for inclusion to LD_LIBRARY_PATH. It would work even without mentioning it here.